app/index.js

```diff
@@ -461,21 +472,31 @@ function downloadImage (imageUrl, filename) {
 }
 
 function getCachedUserInfo () {
-  logger.debug('-----> Inside getCachedUserInfo')
-  const cachedProfile = electronLocalStorage.get('profile')
-  logger.debug(`-----> [${cachedProfile.status}] cachedProfile is ${cachedProfile.data}`)
-  const cachedToken = electronLocalStorage.get('token')
-  logger.debug(`-----> [${cachedToken.status}] cachedToken is ${cachedToken.data}`)
-
-  if (cachedProfile.status && cachedToken.status) {
-    return {
-      token: cachedToken.data,
-      profile: cachedProfile.data,
-      image: electronLocalStorage.get('image').data
+  return new Promise((resolve, reject) => {
+    logger.debug('-----> Inside getCachedUserInfo')
+    const cachedProfile = electronLocalStorage.get('profile')
+    logger.debug(`-----> [${cachedProfile.status}] cachedProfile is ${cachedProfile.data}`)
+    const cachedImage = electronLocalStorage.get('image')
+    logger.debug(`-----> [${cachedImage.status}] cachedImage is ${cachedImage.data}`)
+
+    if (cachedProfile.status && cachedImage.status) {
```

Comment:
Only the profile cache and token cache are required for launching. The image cache is optional. Given there is a relatively higher failure rate for image caching, this check will propagate this higher failure rate to the app launching.

Reply:
Yah, I pushed a commit to remove `cachedImage.status`